sessions-core/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
Noticed that most of these files are empty. We might as well remove them, to decrease some clutter

Reply:
@caiofaustino are these `proguard-rules.pro` files needed for anything? Or just the `consumer-rules.pro`?

Reply:
Yeah those will only be used if we decide to ship the SDK already obfuscated, but I don't think that will ever be the case. So feel free to remove them of you want.